migrations/2019-02-18T214320Z_init/up.sql

```diff
@@ -163,6 +163,17 @@ CREATE TABLE usr_policy (
     PRIMARY KEY(usr_id, policy_id)
 );
 
+CREATE TABLE client (
+    id serial PRIMARY KEY, -- arborist only---not fence
+    client_id text UNIQUE NOT NULL -- SHARED with fence
+);
+
+CREATE TABLE client_policy (
+    client_dbid integer REFERENCES client(id) ON DELETE CASCADE,
```

Comment:
is this not named `client_id` in order to distinguish it from `client.client_id`? maybe the latter could be `fence_id` or something instead

Reply:
Yeah that makes sense, renamed to `fence_client_id` now.